src/style.css

```diff
@@ -1 +1,42 @@
 /* add styles here */
+
+.page__body {
+  margin: 0;
+  gap: 0;
+  opacity: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+/* #endregion */
+
+/* #region stars--1-5 */
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+/* #endregion */
+
+/* #region margins-zero */
+/* stylelint-disable-next-line no-descending-specificity */
+.stars .stars__star:first-of-type {
+  margin-left: 4px;
```

Comment:
Adding a left margin to the first star might not be the best approach if you want to add some space between the stars. Instead, consider adding a right margin to all stars and then removing it from the last one. This is a code quality issue.